airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -55,19 +55,19 @@ definitions:
       - data
       - emitted_at
     properties:
+      namespace:
+        description: "namespace the data is associated with"
+        type: string
       stream:
-        description: "the name of this record's stream"
+        description: "stream the data is associated with"
```

Comment:
why not use the descriptions from the md doc above? I think it's best if we keep those detailed descriptions here rather than duplicate them outside the source of truth

Reply:
next iteration we can clean this up.